uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/BlockIo2/BlackBoxTest/BlockIo2BBTestFunction.c

```diff
@@ -4609,8 +4609,9 @@ BBTestFushBlocksExFunctionAutoTestCheckpoint1(
   EFI_BLOCK_IO2_PROTOCOL                *BlockIo2
   )
 {
+  UINTN                                BlockIo2TokenBufferSize = 4;

```

Comment:
I would prefer to declare a new preprocessor macro (#define) instead of a variable if the value won't be changed.